examples/1.initialisation.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Client initialization code example
+ */
+
+/* eslint-disable no-console */
+import { Networks, BluefinClient, ExtendedNetwork } from "../index";
+
+async function main() {
+  const dummyAccountKey =
+    "royal reopen journey royal enlist vote core cluster shield slush hill sample";
+
+  // using predefined network
+  const client = new BluefinClient(
+    true,
+    Networks.TESTNET_SUI,
+    dummyAccountKey,
+    "ED25519"
```

Comment:
I think we should not pass this parameter and just put a comment that BluefinClient uses Secp256k1 by default but also supports ED25519

Sth like this

```
const client = new BluefinClient(
    true,
    Networks.TESTNET_SUI,
    dummyAccountKey,
) // supports Secp256k1 and ED25519 signature scheme. Default is Secp256k1
```